This fixes the following build errors with the gtk+ faq and tutorial
documentation:

mv: cannot stat `gtk-faq': No such file or directory
mv: cannot stat `gtk-tut': No such file or directory

Signed-off-by: Scott Garman <scott.a.garman@intel.com>
diff -urN gtk+-2.22.1.orig//docs/faq/Makefile.am gtk+-2.22.1/docs/faq/Makefile.am
--- gtk+-2.22.1.orig//docs/faq/Makefile.am	2010-11-15 04:13:09.000000000 -0800
+++ gtk+-2.22.1/docs/faq/Makefile.am	2011-02-17 13:16:30.603214103 -0800
@@ -9,7 +9,8 @@
 	  (cd $(srcdir); \
 	   db2html gtk-faq.sgml; \
 	   test -d html && rm -r html; \
-	   mv gtk-faq html); \
+	   mkdir html; \
+	   mv *.html html); \
 	fi
 
 pdf:
diff -urN gtk+-2.22.1.orig//docs/tutorial/Makefile.am gtk+-2.22.1/docs/tutorial/Makefile.am
--- gtk+-2.22.1.orig//docs/tutorial/Makefile.am	2010-11-15 04:13:09.000000000 -0800
+++ gtk+-2.22.1/docs/tutorial/Makefile.am	2011-02-17 13:15:40.110369180 -0800
@@ -42,7 +42,8 @@
 	  (cd $(srcdir); \
 	   db2html gtk-tut.sgml; \
 	   test -d html && rm -r html; \
-	   mv gtk-tut html; \
+	   mkdir html; \
+	   mv *.html html; \
 	   mkdir html/images; \
 	   cp images/*.png html/images); \
 	fi
